Automattic/simplenote-electron
# Update display of logo for Windows start menu tile

### Fix

Fixes #2935

Currently, if you install the appx or Windows store version of Simplenote and then pin it to the start menu the tile generated has a large logo and doesn't show the app name at all. This is different from most other apps installed.

This shrinks the size of the logo and displays the name on the tile. 

Before:
<img width="402" alt="Screen Shot 2021-06-24 at 10 32 48 AM" src="https://user-images.githubusercontent.com/1326294/123271806-92bf4d80-d4d7-11eb-8778-4cb7ab3447fa.png">

After:
<img width="406" alt="Screen Shot 2021-06-24 at 10 31 54 AM" src="https://user-images.githubusercontent.com/1326294/123271827-97840180-d4d7-11eb-9eb2-d90e1426feef.png">

### Test

1. Install the appx version of SImplenote
2. Pin the app to the start menu
3. Ensure things look proper with a smaller icon and the app title

### Release

- Updated the logo used when pinning to the Windows start menu.
